rmf_robot_sim_ignition_plugins/src/slotcar.cpp

```diff
@@ -11,6 +11,7 @@
 #include <ignition/gazebo/components/Pose.hh>
 #include <ignition/gazebo/components/Static.hh>
 #include <ignition/gazebo/components/AxisAlignedBox.hh>
+#include <ignition/gazebo/components/PoseCmd.hh>
```

Comment:
It seems that PoseCmd components are not currently used so this can be probably removed?

Reply:
removed a549f8f